ios/mapbox_maps_flutter/Sources/mapbox_maps_flutter/Classes/Extensions.swift

```diff
@@ -13,6 +13,29 @@ extension FlutterError: Error { }
 
 // FLT to Mapbox
 
+extension PerformanceStatisticsOptions {
+    func toPerformanceStatisticsOptions() -> MapboxMaps.PerformanceStatisticsOptions {
+        let samplerOptions = samplerOptions
+            .map(\.samplerOption)
+            .reduce(into: MapboxMaps.PerformanceStatisticsOptions.SamplerOptions(), { partialResult, option in
+                partialResult.insert(option)
+            })
+
+        return .init(samplerOptions, samplingDurationMillis: samplingDurationMillis)
+    }
+}
+
+extension PerformanceSamplerOptions {
+    var samplerOption: MapboxMaps.PerformanceStatisticsOptions.SamplerOptions {
```

Comment:
I'd suggest plural here and match with all other local variable of options in our codebase (?!)
```suggestion
    var samplerOptions: MapboxMaps.PerformanceStatisticsOptions.SamplerOptions {
```